packages/client/src/views/CorrectionForm/utils.ts

```diff
@@ -99,6 +100,28 @@ export function isCorrection(declaration: IDeclaration) {
   )
 }
 
+export function motherDoesNotExistAndStateIsMother(
+  declaration: IDeclaration,
+  activeState: string
+) {
+  return (
+    !Boolean(declaration.data.mother.detailsExist) &&
```

Comment:
The tests are failing here:
https://github.com/opencrvs/opencrvs-core/actions/runs/3053534668/jobs/4935260797#step:7:32 

https://github.com/opencrvs/opencrvs-core/pull/3924#discussion_r970410095